policyengine_us/variables/gov/states/mt/tax/income/credits/mt_elderly_homeowner_or_renter/mt_elderly_homeowner_or_renter_credit_eligible.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class mt_elderly_homeowner_or_renter_credit_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Montana Elderly Homeowner/Renter Credit"
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mt.tax.income.credits.elderly_homeowner_or_renter
+        age_head = tax_unit("age_head", period)
```

Comment:
use `older_age_head_spouse` to simplify this, i.e. `return older_age_head_spouse >= p.age_threshold`

Reply:
is this correct? Or do you want me to create a new variable `older_age_head_spouse`? 

<img width="846" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/87988673/a9852d02-bd7a-4df7-a6f4-427d622b1bb6">

Reply:
@MaxGhenis

Reply:
there's already a variable in the system

Reply:
got it.